CMakeLists.txt

```diff
@@ -442,6 +443,26 @@ set (UPM_COMMON_HEADER_DIRS ${CMAKE_HOME_DIRECTORY}/include)
 # Generate a build-only C++ header to add functionality to SWIG'ed modules
 configure_file (${PROJECT_SOURCE_DIR}/cmake/modules/version.hpp.in ${PROJECT_BINARY_DIR}/src/version.hpp @ONLY)
 
+# Cordova binding
+if (BUILDCORDOVABINDING)
+  if (NOT BUILDSWIGJAVA)
+    message(FATAL_ERROR "Cordova bindings needs swig java modules to be built first")
+  endif()
+
+  find_package (Node REQUIRED)
+  find_package (Npm REQUIRED)
+  find_package (Cordova REQUIRED)
+
+  add_custom_target(cordova_binding ALL
```

Comment:
Does this custom target imply sudo access (I see the -h in there).  If so, this isn't a good idea for 'make cordova_binding'

Reply:
No, it doesn't need sudo permission.

Reply:
It appears that running make w/o sudo ends in permission errors for me (since I installed upm-cordova-binding w/-g).

```
[ 53%] Building cordova bindings based on swig java modules
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules/upm-cordova-binding
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules
/usr/local/lib
└── upm-cordova-binding@0.2.0

npm ERR! Linux 4.4.0-97-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "upm-cordova-binding"
npm ERR! node v6.12.0
npm ERR! npm  v3.10.10
npm ERR! path /usr/local/lib/node_modules/upm-cordova-binding
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access

npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules/upm-cordova-binding'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, access '/usr/local/lib/node_modules/upm-cordova-binding'
npm ERR!     at Error (native)
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules/upm-cordova-binding' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /export/users/neck/upm_cordova/build/src/npm-debug.log
```